requirements.txt

```diff
@@ -1,4 +1,4 @@
-axiprop @ https://github.com/hightower8083/axiprop/archive/refs/heads/main.zip
+axiprop==0.5
```

Comment:
If you like, we can also keep the version open (at latest) and fix breaking changes as the occur :)

Reply:
in principle sure, but let me handle this when breaking change occurs -- for now let's keep it fixed against any stupid actions from me =) this version it already supports #126, so next time i'll update if we decide to enable direct propagation of z-defined field